.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: build
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7]
```

Comment:
Why do we run this on multiple versions? (afaict)

Reply:
I find it's helpful to check multiple versions to be sure we're not using some feature that only works in a particular version of Python. This will probably matter more if/when we convert more of FNPRMS to Python and make it easier to run locally.

Reply:
I'm generally more in the "simplify requirements" camp because I think it's less work in the long run: i.e. only the newest version of Python is supported (3.7) and if you're having issues on an older version, then you need to upgrade. But I don't know what the python ecosystem looks like and if that's a reasonable assertion here (e.g. some linux distros are LTS and running older versions) – I'll leave it up to you!

Reply:
Python 3 is pretty stable, so I've never had too much difficulty maintaining compatibility across versions. I'm leaning towards keeping it easy to install and run this locally without too much fiddling around with your environment. The way I see it is we target a slightly-older Python (probably 3.6, since that's what's on the latest Ubuntu LTS release as default) and then just make sure nothing broke on us in newer Pythons.